packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -63,7 +63,12 @@ const Redirect = observer(() => {
             break;
         }
         case 'request_email': {
-            toggleResetEmailModal(true);
+            if (!is_logging_in && !is_logged_in) {
```

Comment:
Should this be an OR condition or AND condition? One is logging_in flag and the other is logged_in flag

Reply:
Yes, it should be `or` but now that i think just checking`!is_logged_in` should be enough. what say? @likhith-deriv

Reply:
We need to add the is_logging_in so that when user logs back they are not redirected back to login page